gateway/ln-gateway/src/lightning/lnd.rs

```diff
@@ -1459,11 +1459,11 @@ mod tests {
         let features_payment_secret = {
             let mut f = Bolt11InvoiceFeatures::empty();
             f.set_payment_secret_optional();
-            WithoutLength(&f).encode()
+            f.encode()
```

Comment:
Is encode equivalent?

Reply:
No longer relevant. I fixed the `WithoutLength` encoding issue.